.github/workflows/ci.yml

```diff
@@ -30,13 +30,25 @@ jobs:
         CHECK_LEAKS: true
       run: ../mspec/bin/mspec -j --timeout 30
 
+    - name: Run specs (Linux / --enable-frozen-string-literal)
+      if: matrix.os == 'ubuntu'
+      env:
+        CHECK_LEAKS: true
+      run: env RUBYOPT="--enable-frozen-string-literal" ../mspec/bin/mspec -j --timeout 30
```

Comment:
Doing it inside the matrix would make the CI about 2x slower.
So I think best to instead have a separate single job on a single version (3.3.0) on ubuntu only.